### DISCUSSION TITLE
CREATE TYPE not showing up anywhere in UI

### DISCUSSION THREAD

**Discussion by irohitb** [supabase/supabase#14543] [Feature Requests]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

In SQL Editor, I created a `Type` but it doesn't show up in the UI anywhere

## To Reproduce

Pasted this in SQL Editor 

```
CREATE TYPE email_invite_at_map AS (
  email VARCHAR,
  invite_at TIMESTAMP
);
```

## Expected behavior

Probably should be able to see it somewhere in the UI

**Comment by saltcod** [comment]
Thanks for this @irohitb. There's still a good few data types we don't yet support in the UI. We hope to eventually support creating and updating all of them. Appreciate the patience.
